src/UI/SummaryListItem.elm

```diff
@@ -0,0 +1,85 @@
+module UI.SummaryListItem exposing (view)
+
+import Element exposing (Element, fill)
+import Element.Keyed as Keyed
+import UI.Badge as Badge exposing (Badge)
+import UI.Palette as Palette exposing (brightnessLight, brightnessLighter, brightnessMiddle, toneGray, tonePrimary)
+import UI.RenderConfig exposing (RenderConfig)
+import UI.Text as Text exposing (ellipsize)
+
+
+type alias ColorsHelper =
+    { badge : String -> Badge
+    , title : Palette.Color
+    , caption : Palette.Color
+    }
+
+
+view : RenderConfig -> Bool -> String -> String -> Int -> Element msg
+view renderConfig isSelected title caption number =
+    let
+        colors =
+            listItemColors isSelected
+    in
+    Keyed.row
+        [ Element.width fill
+        , Element.paddingEach { top = 11, bottom = 11, left = 20, right = 12 }
+        ]
+        [ ( "label", listItemLabel renderConfig colors title caption )
+        , ( "badge", listItemBadge renderConfig colors.badge number )
```

Comment:
:sweat_smile:  Hey man, could we change `Int` to be a `Badge` instead, so we can color them and insert text? It will be necessary for Phase 3!!